_posts/ianmoore/2016-03-22-natalia-and-ian-file.md

```diff
@@ -0,0 +1,22 @@
+---
+layout: post
+author: touchwick
+title: "Ian and Natalia's File Exercises"
+---
+This is exercise 1, which Ian completed:
+
+<iframe src="https://trinket.io/embed/python3/a0ab418311" width="100%" height="356" frameborder="0" marginwidth="0" marginheight="0" allowfullscreen></iframe>
+
+This is exercise 2, which Ian was unable to get to work successfully:
+
+<iframe src="https://trinket.io/embed/python/b5a789e422" width="100%" height="600" frameborder="0" marginwidth="0" marginheight="0" allowfullscreen></iframe>
+
+Exercise 2 went through a very long revision process, but ultimately I could not get it to recognize the phrase that precedes the numbers needed to compile the average. I understand how to build the average once its components are assembled, but I'm not sure how to get it to work now. Any suggestions on this would be greatly appreciated, because I am thoroughly stumped.
+
+This is exercise 3, which Natalia completed:
+
+<iframe src="https://trinket.io/embed/python/07fc8d5644" width="100%" height="600" frameborder="0" marginwidth="0" marginheight="0" allowfullscreen></iframe>
+
+And this is exercise 4, which Natalia also completed:
+
+<iframe src="https://trinket.io/embed/python3/b6682a1642" width="100%" height="356" frameborder="0" marginwidth="0" marginheight="0" allowfullscreen></iframe>
```

Comment:
@batlopez I don't see your work embedded here in exercise 4.  Is this the correct link?  Did you remix and press save?